lib/dependency_spy.rb

```diff
@@ -65,8 +72,9 @@ def self.check(path = Dir.pwd, files = nil, platform = nil, database_path = YAVD
             vulnerable = vuln.vulnerable_versions ? vuln.vulnerable_versions.any? { |vv| DependencySpy::SemVer.intersects(vv, version) } : false
             unaffected = vuln.unaffected_versions ? vuln.unaffected_versions.any? { |vu| DependencySpy::SemVer.intersects(vu, version) } : false
             patched    = vuln.patched_versions ? vuln.patched_versions.any? { |vp| DependencySpy::SemVer.intersects(vp, version) } : false
+            ignored    = ignore.include?(vuln.id)
 
-            if unaffected || patched
+            if unaffected || patched || ignored
```

Comment:
Logging this with the debug flag could be nice.

Reply:
Hey @rtfpessoa I'm not sure if I completely understand the suggested changes? Can you please elaborate a little bit more so that I can make the write changes. Thanks.

Reply:
There is a `verbose` parameter you can pass to the CLI. It would be nice to log a message here in case verbose is enabled stating that one vulnerability was found but ignored.